[tune] Add test for multi-tenancy workaround and documentation to FAQ (#32560)
ray-project/ray: doc/source/ray-air/user-guides.rst

- **RAY_AIR_FULL_TRACEBACKS**: If set to 1, will print full tracebacks for training functions,
  including internal code paths. Otherwise, abbreviated tracebacks that only show user code
  are printed. Defaults to 0 (disabled).

.. _air-multi-tenancy:

> Running multiple AIR jobs concurrently on one cluster

Reviewer: ```suggestion
Running multiple AIR jobs concurrently on a single cluster
```

-----------------------------------------------------
Running multiple AIR training or tuning jobs at the same

> time on one cluster is not officially supported.

Reviewer: ```suggestion
time on a single cluster is not officially supported.
```

> This means that we don't test this workflow

Reviewer: ```suggestion
We don't test this workflow
```

> and generally encourage to use multiple smaller clusters

Reviewer: ```suggestion
and recommend the use of multiple smaller clusters
```

instead.

> If you still want to do this, please refer to

Reviewer: ```suggestion
If you still want to do this, please refer to
```
```suggestion
If you still want to do this, refer to
```
